data/getdata.sh

```diff
@@ -7,8 +7,8 @@ echo "Unzipping the data files (overwrites any existing .tsv files)..."
 gunzip -vf *.gz
 echo
 echo "Removing IMDb '\N' values..."
-sed --in-place 's/\\N//g' *.tsv
+ed -s *.tsv <<< 's/\\N//g'
 echo
 echo "IMDb data files ready:"
 ls *.tsv
-echo 
+echo
```

Comment:
Just a minor comment that I see too many "echo " for new lines

Reply:
I think having the new lines breaks up the wall of text that can happen. However, if folks find it distracting more than useful, I can remove them.

Reply:
I prefer a `printf "\n\n\n"` with however many newlines you want, that way the echo echo echo echo etc doesn't look so distracting